infra/docker-swarm/ansible/setup.yaml

```diff
@@ -0,0 +1,77 @@
+- hosts: all
+  become: true
+  gather_facts: false
+  roles:
+  # Prepare system to be managed by Ansible.
+  - role: robertdebock.bootstrap
+    bootstrap_user: "{{ ansible_user }}"
+    bootstrap_retries: 3
+    bootstrap_wait_for_host: yes
+    tags: [bootstrap]
+  # Set hostname
+  - role: setup
+    tags: [setup]
+  # Docker
+  - role: geerlingguy.docker
+    tags: [docker]
+  - role: geerlingguy.pip
+    pip_install_packages:
+    - name: docker
+    tags: [pip]
+  - role: docker-cleanup
+    tags: [docker-cleanup]
+  # Filebeat
+  - role: filebeat
+    tags: [filebeat]
+  # Log rotation
+  - role: nickhammond.logrotate
+    logrotate_scripts:
+    - name: docker-containers
+      path: /var/lib/docker/containers/*/*.log
+      options:
+        - daily
```

Comment:
The rotation may interfere with Filebeat. Have we investigated this?

Reply:
This seems to be a non-issue for now. Let's deal with this problem, if it comes up.

Reply:
Actually @nonsense, I disagree with deferring. If this shows up, it'll show up as "logs missing". And it'll send us down a painful debugging trip which can last for hours or days. 10 minutes well spent now investigating this can save us from a lot of pain.